addons/nightvision/functions/fnc_onFiredPlayer.sqf

```diff
@@ -55,12 +55,7 @@ TRACE_1("final", _visibleFire);
 if (_visibleFire <= 1.5) exitWith {};
 if ((random (linearConversion [1, 4, GVAR(nvgGeneration), 10, 20])) > _visibleFire) exitWith {};
 
-GVAR(ppEffectCCMuzzleFlash) = ppEffectCreate ["ColorCorrections", 1237];
-GVAR(ppEffectCCMuzzleFlash) ppEffectEnable true;
-GVAR(ppEffectCCMuzzleFlash) ppEffectForceInNVG true;
-
-GVAR(ppEffectCCMuzzleFlash) ppEffectAdjust [1, 1, -1, [0, 0, 0, 0], [0, 0, 0, 1], [0, 0, 0, 1]];
-GVAR(ppEffectCCMuzzleFlash) ppEffectCommit 0;
-
-GVAR(ppEffectCCMuzzleFlash) ppEffectAdjust [1, 1, 0, [0, 0, 0, 0], [0, 0, 0, 1], [0, 0, 0, 1]];
-GVAR(ppEffectCCMuzzleFlash) ppEffectCommit 0.07;
+QGVAR(cutoff) cutText ["", "BLACK", .1];
```

Comment:
If we wanted to make it slightly more realistic perhaps we could make use of `WHITE OUT` here in the `cutText`

Reply:
I should also point out in my experience that the `speed` parameter listed on biki is actually more like the length of time it's shown for

Reply:
`"test" cutText ["", "BLACK", 5];` will fade from transparent to black over 5 second and stay that way

Reply:
Ah okay, I guess it's length of time when it's `"PLAIN"`